automatic/firefox/README.md

```diff
@@ -17,7 +17,22 @@ Bringing together all kinds of awesomeness to make browsing better for you.
 
 ## Package Parameters
 
-- `l=<locale>` - Install given Firefox locale. For example `choco install Firefox --params "l=en-GB"`. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+- `l=<locale>` - Install given Firefox locale. See the [official page](https://releases.mozilla.org/pub/firefox/releases/latest/README.txt) for a complete list of available locales.
+
+Command-line options for installer configuration. See the [official page](https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/FullConfig.html) for details and defaults.
+- `/InstallDirectoryPath=<path>`
+- `/InstallDirectoryName=<path>`
```

Comment:
`/InstallDir` is used by  other packages and choco enterprise.

Reply:
OK, I used the options as provided by Firefox. I can change it to `/InstallDir` and map it to `/InstallDirectoryPath`.

Reply:
You can also support both if you like.

Reply:
Maybe, better:

InstallDir = Path\Name

So we have 3 options here. One combinational and standard one, other two are regular FF options. Everybody happy.

Reply:
If you do it that way, you can't use all options together.

InstallDir - ignore Path\Name because it contains it.
No installDir, provide Path and/or Name.

Reply:
Not sure I got you right. By using `/InstallDir:path\name` mapped to `/InstallDirectoryPath=` it should be possible to install anywhere you want.
The installer option `/InstallDirectoryName=` is simply setting the name within the ProgramFiles folder. So this can be ignored as it's possible to accomplish the same with `/InstallDir:path\name`